packages/manager/src/features/Linodes/LinodesDetail/LinodeNetworking/NetworkingSummaryPanel/NetworkingSummaryPanel.tsx

```diff
@@ -16,23 +17,29 @@ interface Props {
 export const LinodeNetworkingSummaryPanel = React.memo((props: Props) => {
   // @todo maybe move this query closer to the consuming component
   const { data: linode } = useLinodeQuery(props.linodeId);
+  const { isGeckoGAEnabled } = useIsGeckoEnabled();
   const theme = useTheme();
 
   if (!linode) {
     return null;
   }
 
+  const hideNetworkTransfer =
+    isGeckoGAEnabled && linode.site_type === 'distributed';
```

Comment:
Could we simplify this to

```suggestion
  const showNetworkTransfer = linode.site_type === 'core';
```

So that the ternary below could be replaced with a `&&`